# VelSensorSetup block
# Block_Number 4244

std_msgs/Header header

# SBF block header including time header
BlockHeader block_header

uint8   port
float32 lever_arm_x
float32 lever_arm_y
float32 lever_arm_z